controller/src/main/java/io/pravega/controller/store/stream/StoreException.java

```diff
@@ -114,12 +134,42 @@ public NodeNotEmptyException() {
     }
 
     /**
+     * Exception type when you are attempting to update a stale value.
+     */
+    public static class WriteConflictException extends StoreException {
```

Comment:
here and elsewhere, retryability trait is lost. we need to retain retryable trait by implementing retryable exceptions wherever it was originally so.